service/src/test/java/uk/gov/hmcts/reform/fpl/validation/validators/time/TimeNotZeroValidatorTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.fpl.validation.validators.time;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.validation.ConstraintViolation;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(SpringExtension.class)
+class TimeNotZeroValidatorTest extends TimeValidatorTest {
+
+    @Test
+    void shouldReturnAnErrorWhenAllTimeFieldsAreZero() {
+        final LocalTime ZERO = LocalTime.of(0, 0, 0);
+        hearingBooking = HearingBooking.builder()
+            .startDate(LocalDateTime.of(FUTURE.toLocalDate(), ZERO))
+            .endDate(FUTURE.plusDays(1))
+            .build();
+
+        final Set<String> violations = validator.validate(hearingBooking, group)
+            .stream()
+            .map(ConstraintViolation::getMessage)
+            .collect(Collectors.toSet());
+
+        assertThat(violations).hasSize(1).containsOnly("Enter a valid start time");
+    }
+
+    @Test
+    void shouldNotReturnAnErrorWhenATimeFieldIsNotZero() {
```

Comment:
`shouldNotReturnAnErrorWhenTimeFieldIsNotZero`

Reply:
`shouldNotReturnAnErrorWhenTimeIsNotZero` probably works as well

Reply:
Yeah I like that!